db/types.go

```diff
@@ -51,6 +51,10 @@ type StreamingParams struct {
 	//
 	// required: true
 	Protocol string `redis-hash:"protocol" json:"protocol"`
+
+	// the playlist file name
+	// required: true
+	PlaylistFileName string `json:"playlistFileName,omitempty"`
```

Comment:
We need to define redis-hash to get it persisted in Redis.

This brings the question: should our tests use Redis instead of memory? I know it makes things more complicated, but we'd be able to catch this in the test suite.

Or should we  just add a test where we persist a whole job and ensure that it can get it back?